qemu/tests/block_hotplug_passthrough.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_disk
+from avocado.utils import process
+from provider.storage_benchmark import generate_instance
+from provider.block_devices_plug import BlockDevicesPlug
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Hotplug/unplug passthrough disk test:
+    1) Create passthrough disk with scsi_debug tool.
+    2) Start the guest.
+    3) Hotplug this passthrough disk.
+    4) Create partition on this disk and format it.
+    5) Do iozone/dd test on this disk.
+    6) Reboot the guest.
+    7) Unplug this passthrough disk.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def create_path_disk():
+        """Create a passthrough disk with scsi_debug """
+        create_cmd = params.get("create_command")
+        process.system_output(create_cmd, shell=True, verbose=False)
```

Comment:
better to `create_cmd = params["create_command"]` since the `create_command` must be set.

Reply:
And suggest passing the `timeout`.

Reply:
updated, thanks~